packages/framework/client-logger/app-insights-logger/src/fluidAppInsightsLogger.ts

```diff
@@ -9,6 +9,56 @@ import {
 	type ITelemetryBaseLogger,
 } from "@fluidframework/core-interfaces";
 
+/**
+ * An Enum of Fluid telemetry event categories
+ */
+export const TelemetryEventCategory = {
+	PERFORMANCE: "performance",
+	GENERIC: "generic",
+	ERROR: "error",
+} as const;
+
+/**
+ * The type for the enum of Fluid telemetry event categories
+ */
+export type TelemetryEventCategory =
+	(typeof TelemetryEventCategory)[keyof typeof TelemetryEventCategory];
+
+/**
+ * The configuration object for the {@link FluidAppInsightsLogger}
+ */
+export interface FluidAppInsightsLoggerConfig {
+	filterConfig: {
```

Comment:
Nit: could we add a very brief comment to this property? The internal prop docs look great, but it would be good to add something brief here too.

Reply:
Added some docs.